index.html

```diff
@@ -36,127 +38,126 @@
-                Drop us a line on your topic on <a href="http://twitter.com/reactberlin">
-                Twitter</a> or <a href="https://github.com/reactjs-berlin/reactjs.berlin">create a pull request on GitHub</a> to this page right away.
-              </p>
-            </div>
-            <div class="speaker">
-              <h3>Free slot</h3>
-              <p class="speaker-name">This slot could be yours!<br />Submit your talk now.</p>
-              <p>
-                You did some great stuff with JavaScript? You want to show it to the community?
-              </p>
-              <p>
-                Drop us a line on your topic on <a href="http://twitter.com/reactberlin">
-                Twitter</a> or <a href="https://github.com/reactjs-berlin/reactjs.berlin">create a pull request on GitHub</a> to this page right away.
-              </p>
-            </div>
-            <div class="speaker">
-              <h3>Free slot</h3>
-              <p class="speaker-name">This slot could be yours!<br />Submit your talk now.</p>
-              <p>
-                You did some great stuff with JavaScript? You want to show it to the community?
-              </p>
-              <p>
-                Drop us a line on your topic on <a href="http://twitter.com/reactberlin">
-                Twitter</a> or <a href="https://github.com/reactjs-berlin/reactjs.berlin">create a pull request on GitHub</a> to this page right away.
-              </p>
-            </div>
-          </div>
-        </section>
+        <div class="clearfix">
+          <p>
+            When we started this event in 2015, it was the first React focused meetup in Berlin and the community 
+            was still young. 
+            Now, some years and many great meetups later, several other similar events around React have evolved. 
+            So we find that running another one just doesn’t make sense anymore.
+          </p>
+        </div>
       </article>
-    </main>
-
-    <footer class="content" role="complimentary">
-      <div class="clearfix three-cols">
-        <div>
-          <h2>Get in touch</h2>
-          <ul>
-            <li>
-              Follow ReactJS.berlin on <a href="http://twitter.com/reactberlin">Twitter</a>
-            </li>
-            <li>
-              Join our <a href="http://www.meetup.com/React-Berlin/">Meetup Group</a>
-            </li>
-            <li>
-              Watch the code on <a href="https://github.com/reactjs-berlin/reactjs.berlin">Github</a>
-            </li>
-          </ul>
+      <div class="clearfix three-cols">  
+        <div class="speaker">
+          <h2>Looking for React meetups?</h2>
+          <p>
+            If you are still looking for React meetups, events or courses, we suggest taking a look at the
+            <a href="https://www.meetup.com/topics/reactjs/de/berlin/?country=de&zipstatecity=berlin&state=&radius=40"
+              title="Berlin's finest JavaScript Usergroup">meetup.com groups.</a>. 
```

Comment:
I removed the `title` attribute, but since then the content has changed, so `ReactJS groups on meetup.com` doesn't fit as much anymore. 
You can see for yourself when we look at the content together.